std/algorithm/comparison.d

```diff
@@ -211,6 +211,13 @@ Throws: If none of the choice matches, a $(D SwitchError) will be thrown.  $(D
 SwitchError) will also be thrown if not all the choices are void and a void
 choice was executed without throwing anything.
 
+Params:
+    choices = The choices to select from.
+    switchObject = The $(D Object) compared to the choices.
```

Comment:
This reads a bit awkwardly. Maybe "The Object to compare each choice with" instead. Also, can't we use backticks instead of $(D ...) now?